ibm/service/contextbasedrestrictions/resource_ibm_cbr_zone.go

```diff
@@ -412,7 +364,9 @@ func resourceIBMCbrZoneUpdate(context context.Context, d *schema.ResourceData, m
 
 	currentZone, response, found, err := getZone(contextBasedRestrictionsClient, context, zoneId)
 	if err != nil {
-		return diag.FromErr(err)
+		tfErr := flex.TerraformErrorf(err, fmt.Sprintf("getZone failed: %s", err.Error()), "ibm_cbr_zone", "update")
```

Comment:
It feels wrong to include the local function (`getZone`) in the error message.

Reply:
Removed getZone failed as the error message returned from getZone already provides sufficient details to describe the error.